web/docs/understand/language/pipelines.md

```diff
@@ -0,0 +1,129 @@
+---
+sidebar_position: 0
+---
+
+# Pipelines
+
+The VAST language centers around one principle: **dataflow pipelines**.
+
+A pipeline is chain of [operators](operators) that represents a flow of data.
+Operators can produce, transform, or consume data. Think of it as UNIX pipes or
+Powershell commands where output from one command is input to the next:
+
+![Pipeline Chaining](pipeline-chaining.excalidraw.svg)
+
+VAST distinguishes three types of operators:
+
+1. **Source**: generates new data
+2. **Transformation**: modifies data
+3. **Sink**: consumes data
```

Comment:
Should this be s/data/events to match the terminology used in other places? That's the element type that the operators are working on.

Reply:
Not in this context. We've introduced this as dataflow, also in the diagram. At this high level, I don't want to add a term that needs to be defined.

Rather, I see this in a different part where talk about the element types.

Reply:
Fine then, I have no strong opinion on this. I would've preferred it, though. :)

Reply:
Of course, you're the person building this. 😉

What's key here is a gentle introduction and leaving implementation aspects to later sections where the distinction matters.